packages/model-viewer/src/test/three-components/gltf-instance/correlated-scene-graph-spec.ts

```diff
@@ -109,11 +111,10 @@ suite('correlated-scene-graph', () => {
 
         let name;
         cloneCorrelatedSceneGraph.threeObjectMap.forEach(
-            (reference, threeObject) => {
+            (mappings, threeObject) => {
               if ((threeObject as MeshStandardMaterial).isMaterial) {
-                name =
-                    cloneCorrelatedSceneGraph.gltf.materials![reference.index]
-                        .name;
+                const index = mappings.materials!;
+                name = cloneCorrelatedSceneGraph.gltf.materials![index].name;
```

Comment:
Good start; are there any end-to-end tests we should add? I recall you saying this was fixing  a bug (or removing a workaround) for loading skinned meshes?

Reply:
Will be adding these soon...